<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyCanvas</class>
 <widget class="QWidget" name="MyCanvas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MyCanvas{
	background-color:  #f8f8f8;
}
QWidget#widget{
	border: 1px solid #BEBFC2;
	border-radius: 2px;	
	background-color:  white;
}
QLineEdit{
   color: #69788F;
	padding:5px;
    background-color: white;
    border: 0px solid #008cba;
}
QLineEdit#queryEdit,#role1Edit,#role2Edit{
   border: 1px solid #BEBFC2;
}
QPlainTextEdit{
   color: #69788F;
	padding:5px;
    background-color: white;
    border: 0px solid #008cba;
}
QPushButton{
	background-color: #E5F1FC;
	border:0px solid white;
	color: #2196F3;
	border-radius: 5px;
}
QPushButton:hover{
	background-color: #D7EAFB;
	color: #158EF2;
}
QLabel{
	color: #69788F;
}
QLabel#imgLabel{
	background-color: #f2f2f2;
}
QGraphicsView{
	border: 1px solid #BEBFC2;
	border-bottom: 0px;
}
QCheckBox::indicator {
        width: 25px;
        height: 25px;
}
QCheckBox::indicator:enabled:unchecked {
        image: url(:/icon/checkbox0.svg);
}
QCheckBox::indicator:enabled:checked {
        image: url(:/icon/checkbox1.svg);
}
QPushButton#queryButton1,#queryButton2{
	border-radius: 0px;	
}
QPushButton#addRole,#exportImg,#debugButton,#resetCanvas{
	background-color: rgba(255,255,255,100);
	border:1px solid #60a2dc;
	color:#2e84cf;
	border-radius: 5px;
}
QPushButton#addRole:hover,#exportImg:hover,#debugButton:hover,#resetCanvas:hover{
	background-color: #D7EAFB;
	color: #158EF2;
}
QPushButton#deleteItem{
	background-color: rgba(255,255,255,100);
	border:1px solid #DE5F5F;
	color:#CF322E;
	border-radius: 5px;
}
QPushButton#deleteItem:hover{
	background-color: #FBD7D7;
}
QTabWidget::pane{
	background-color: white;
	border:0px solid white;
	font:13pt '微软雅黑';
}
QTabBar::tab{
    color: #69788F;
	padding: 2px;
	margin: 0 20px; 
	background-color: white;
}
QTabBar::tab:selected{
    color: #2196F3;
	border-bottom: 2px solid #2196F3;
}
QTabWidget::pane{
	border-top: 1px solid #DADBDC;
}
 QComboBox{
	padding-left: 10px;
    color: #444;
    background-color: white;
    border: 1px solid #BEBFC2;
	border-radius: 5px;
}
 QComboBox:hover{
	background: #f6f6f6;
}
/* 下拉后，整个下拉窗体样式 */
QComboBox QAbstractItemView {
    outline: 0px solid gray;   /* 选定项的虚框 */
    border: 1px solid  #BEBFC2;  /* 整个下拉窗体的边框 */
    color:  #444;
    background-color: white;  /* 整个下拉窗体的背景色 */
	selection-color: #2196F3;
    selection-background-color: #D7EAFB;  /* 整个下拉窗体被选中项的背景色 */
}
/*下拉按钮样式*/
QComboBox::drop-down {
	width:20px; 
	border: 0;
	padding: 5px;
} 
QComboBox QAbstractItemView::item { 
	height: 40px;
	font-size: 10px;
}

QComboBox QAbstractItemView::item:hover {
	background-color: #D7EAFB;
	color: #158EF2;
}
/*下拉箭头样式*/
QComboBox::down-arrow { 
	width: 20px;
	height: 20px;
	color: #BEBFC2;
	border-image: url(:/icon/004.ico);
;}
QPushButton#blueBtn{
	background-color: #73A5D2;
}
QPushButton#blueBtn:hover{
	background-color: #92CBF5;
}
QPushButton#redBtn{
	background-color: #D88785;
}
QPushButton#redBtn:hover{
	background-color: #F79286;
}
QPushButton#purpleBtn{
	background-color: #A482D2;
}
QPushButton#purpleBtn:hover{
	background-color: #C8A8F9;
}
QPushButton#yellowBtn{
	background-color: #D0B575;
}
QPushButton#yellowBtn:hover{
	background-color:#F5D68B;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="addRole">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>添加人物</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteItem">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>移除选中项</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportImg">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>导出图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="debugButton">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>保存文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetCanvas">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>清空画布</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>力导向分布优化</string>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6" stretch="20,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="zoomLabel">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid #BEBFC2;
border-top: 0px;
border-right: 0px;
background-color: white;
padding: 5px;</string>
             </property>
             <property name="text">
              <string>缩放比例：100%</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="countLabel">
             <property name="font">
              <font>
               <family>Microsoft YaHei UI</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid #BEBFC2;
border-top: 0px;
border-left: 0px;
background-color: white;
padding: 5px;</string>
             </property>
             <property name="text">
              <string>Nodes：0，Edges：0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,7,0,3,1">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>按姓名查询</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>按ID查询</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Tab 1</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="queryEdit">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Microsoft YaHei</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="queryButton1">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Microsoft YaHei</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>查询</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="spacing">
               <number>5</number>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="role1Edit">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Microsoft YaHei</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Microsoft YaHei</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>和</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="role2Edit">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Microsoft YaHei</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="queryButton2">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Microsoft YaHei</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>查询</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="imgLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>-1</number>
            </property>
            <property name="openExternalLinks">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openFile">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 0px;</string>
            </property>
            <property name="text">
             <string>···</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="idLabel">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>ID:00</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nameEdit">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Null</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="infoEdit">
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:15px;
text-align: center;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string>No description</string>
          </property>
          <property name="backgroundVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="setColor">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>设置颜色：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>蓝色</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>红色</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>紫色</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>黄色</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>洋红</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>绿色</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>灰色</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>确定</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
